histolab/masks.py

```diff
@@ -178,7 +188,15 @@ def _mask(self, slide) -> np.ndarray:
             Binary mask of the tissue area. The dimensions are those of the thumbnail.
         """
         thumb = slide.thumbnail
-        filters = FiltersComposition(histolab.slide.Slide).tissue_mask_filters
+
+        # Generate appropiate composition of filter
+        filters = (
+            FiltersComposition(histolab.slide.Slide)
+            if len(self.custom_filters) == 0
+            else FiltersComposition(
+                histolab.filters.image_filters.Compose, *self.custom_filters
+            )
+        ).tissue_mask_filters
```

Comment:
Sorry but I see some problems in this portion of code:
1. `histolab.filters.image_filters.Compose` 👉🏽 could you import `Compose` without the whole reference of the histolab path?
2. The statement is too much convoluted maybe you should split it in something more readable (remember that the zen of python says: `beautiful is better than ugly 😂 `